index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mens Portfolio</title>
+    <link rel="stylesheet" href="css/index.css">
+</head>
+<body>
+
+<div class="flex_container">
+    <nav class="flex_nav">
+        <a href="" class="toggleNav"> 
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+        </a>
+        <ul>
+            <li class=""><a href="#">Hello</a></li>
+            <li class=""><a href="#">Portfolio</a></li>
+            <li class=""><a href="#">About</a></li>
+            <li class=""><a href="#">Contact</a></li> 
+        </ul>
+    </nav>
+
+    <div class="mobile_logo">
+        <img src="img/mobile_logo.png" alt="">
+    </div>
+
+
+    <div class="text_class">
+        Emmanuel Mensah
+    </div>
+
+    <div class="text_subject">
+        Hello!I am a software developer! I can 
+        help you build a product, feature or website.
+        Take a look of my works. If you like what you see 
+        and have a project you need coded, don't hesitate 
+        and contact me.
+    </div>
+
+    <div class="text_button">
+       <p>Start Collaboration</p> 
+    </div>
+
+
+    <div class="mobile_image">
+        <img src="img/mobile_image.png" alt="">
+    </div>
+
+

```

Comment:
Kindly wrap the headline section (the second section content which is below the `header` tag) inside the `section` tag as its the project's requirement. That makes the code more readable and clean.